bin/yk-config

```diff
@@ -10,22 +10,59 @@ if [ ! -d "${DIR}/../ykrt" ]; then
     exit 1
 fi
 
-# The post-link time optimisations that we can tolerate. At this moment this is
-# only the instcombine pass in the below configuration (copied over from the
-# default pipeline). Using `instcombine` on its own breaks, possibly due to the
-# missing `no-verify-fixpoint`.
-POSTLINK_PASSES_STR="instcombine<max-iterations=1;no-use-loop-info;no-verify-fixpoint>"
+set_aot_pipeline() {
+    level=$1
+    # env var always takes precident.
+    if [ ! -z "${YKB_AOT_OPTLEVEL}" ]; then
```

Comment:
I also wasn't 100% certain that this should be a `YKB`. Yes, it's for building stuff, but for building the **interpreter**, as opposed to yk components.